client/components/language-picker/modal.scss

```diff
@@ -27,8 +27,8 @@
 	.language-picker-modal__body {
 		overflow-y: auto;
 		// Fix cropped box shadow.
-		padding-left:1px;
-		margin-left:-1px;
+		padding:1px;
+		margin:-1px;
 	}
```

Comment:
Can we pls add an image in the PR and detail what these changes aim to achieve? Ideally, it would be great if this can be removed altogether. Is it something that all modal usages will require or due to other content forcing the clipping?

Someone bumping into this in the future may look to the PR introducing the change. It doesn't say much.

Reply:
Thanks, I've added the before and after. 

The fix is particular to this situation, where we have an input with `box-shadow` inside a container using `overflow-y:auto`. 

As `overflow-y:auto` clips at the container padding level and `box-shadow` is not considered in the width calculations, this results in the before-image situation.

AFAIK, there's no cleaner way to handle box-shadow clipping, but I'm open to suggestions.

Reply:
Thanks. Indeed. The padding/margin hack moves the boundary. There's still some clipping, though, with other elements e.g. when selecting one of the regions:

<img width="500" alt="Screenshot 2025-05-22 at 12 48 41 PM" src="https://github.com/user-attachments/assets/ae83454d-b67e-47f8-9967-45c9124f097a" />

The other thing I notice is how the internal buttons touch when both selected in a row:

<img width="500" alt="Screenshot 2025-05-22 at 12 49 08 PM" src="https://github.com/user-attachments/assets/734fe087-3cf8-4f12-a589-12e49284c784" />

Separate issues, but perhaps if we can fix the paddings a little better across body/header/footer, we might also be able to avoid the padding/margin hack?